test/unit/vdom/elements.js

```diff
@@ -136,49 +135,6 @@ describe('vdom/elements', () => {
     });
   });
 
-  it('slot', (done) => {
-    const elem1 = new (element().skate({
-      render () {
-        vdom.element('slot', { name: 'test' });
-      }
-    }))();
-    const elem2 = new (element().skate({
-      render () {
-        vdom.element('slot', { name: 'test' });
-      }
-    }))();
-
-    fixture().appendChild(elem1);
-    fixture().appendChild(elem2);
-
-    afterMutations(
-      () => {}, // .render()
-      () => {
-        const ch1 = elem1[symbols.shadowRoot].firstElementChild;
-        const ch2 = elem2[symbols.shadowRoot].firstElementChild;
-
-        function assertSlotElement () {
-          expect(ch1.tagName).to.equal('SLOT', 'vdom');
-          expect(ch1.getAttribute('name')).to.equal('test', 'vdom');
-          expect(ch2.tagName).to.equal('SLOT', 'vdom.element(slot)');
-          expect(ch2.getAttribute('name')).to.equal('test', 'vdom.element(slot)');
-        }
-
-        if (shadowDomV1) {
-          assertSlotElement();
-        } else if (shadowDomV0) {
```

Comment:
We've removed support for no shadow DOM, so the only thing we need to test now is SD v1.